tsconfig.json

```diff
@@ -1,5 +1,13 @@
 {
   "compilerOptions": {
       "lib": ["es2017", "dom"],
-  }
+      "resolveJsonModule": true,
+      "outDir": "dist/"
+  },
+  "include": [
+    "src/branch-manager-service/task-service/presubmit-pr.ts",
```

Comment:
How about having an `index` file for the `branch-manager-service/` directory and just export all requires files there?